src/interface_r/R/auto_generated_wrappers.R

```diff
@@ -1,8 +1,10 @@
 # -------------------------------------------------------------------------------
 #             *** WARNING: DO NOT MODIFY THIS FILE ***
 #              
-# Instead, modify scripts/gen_wrappers.R which automatically generates this file
-# ------------------------------------------------------------------------------- 
+# Instead, modify scripts/gen_wrappers.R which automatically generates this file.
+#
+# reticulate version: 1.6.0
```

Comment:
I would just changed the wording to: `Generated by reticulate version: ...`

Reply:
Thanks! Changed.